packages/communication-react/review/communication-react.api.md

```diff
@@ -711,6 +711,7 @@ export type CallWithChatCompositeIcons = {
 // @beta
 export type CallWithChatCompositeOptions = {
     callControls?: boolean | CallWithChatControlOptions;
+    mobilePane?: MobilePaneOptions;
```

Comment:
I am curious now that we have an adaptive UI (e.g we different mobile and non mobile modes) should we just separate out options specific for mobile?

e.g

{
  ...
  mobile : {
      ...mobileSpecificOptions
   }
}

Reply:
given isMobile is an existing prop we should actually probably avoid anything `mobile*` here to avoid prop overlap

Instead one idea here would be to make this something like: 
```
browserOverrides: {
  closeSidePaneOnBackButton: boolean;
}
```

then a developer would do:
```
options={{
  browserOverrides: {
    closeSidePaneOnBackButton: isUserOnMobile() ? true : false
  }
}}
```

It would mean we have to set as `false` by default for mobile (as desktop should likely always be false) so less out-of-box set up